src/components/back-to-top/_macro.spec.js

```diff
@@ -0,0 +1,60 @@
+/** @jest-environment jsdom */
+
+import * as cheerio from 'cheerio';
+
+import axe from '../../tests/helpers/axe';
+import { renderComponent, templateFaker } from '../../tests/helpers/rendering';
```

Comment:
Import not being used

```suggestion
import { renderComponent } from '../../tests/helpers/rendering';
```